thumbor/Dockerfile

```diff
@@ -19,20 +19,21 @@ ENV SHELL bash
 ENV WORKON_HOME /usr/src/app
 WORKDIR /usr/src/app
 
-ARG DOCKERHOST=172.17.0.1
-ENV DOCKERHOST ${DOCKERHOST}
 COPY requirements.txt /usr/src/app/requirements.txt
-RUN pip install --trusted-host None --no-cache-dir --use-wheel \
-   --extra-index-url http://${DOCKERHOST}:9009/simple/ \
-   --trusted-host ${DOCKERHOST} \
+RUN pip install -U pip==9.0.3
```

Comment:
if this is pinned to v9, then not necessary to install patched tornado-botocore, right?

Reply:
Yes :) I wasn't sure which option to choose to be honest, so left both. Which is a bit silly, but at least we're having this conversation. What's the best option in your opinion?

Reply:
:man_shrugging: not sure. Let's stick with pip 9, I guess. Adding a comment explaining would be great too.